### PR TITLE
chip color 

### PR DESCRIPTION
**Pull request by enavhasson**
change chip color

### CODE REVIEW

File: app/src/main/res/layout/filter_dialog.xml [hunk 1]
```diff
@@ -52,7 +52,7 @@
             android:layout_height="wrap_content"
             app:chipMinTouchTargetSize="0dp"
             app:chipSpacingHorizontal="10dp">
-
+Z
```
Comment (zoharHenMicrosoft) [inline app/src/main/res/layout/filter_dialog.xml:55]: by mistake?

File: app/src/main/java/com/example/cocktails/CustomItem/UserItemLevel1.kt [hunk 2]
```diff
@@ -45,18 +47,24 @@ class UserItemLevel1 : AppCompatActivity() {
     val CATEGORY_KEY = "category"
     val ICON_KEY = "icon"
     val UPLOAD_IMG_KEY = "upload_img"
+    val YES= "yes"
+    val NO ="no"
+    val CANCEL="cancel"
+    val DIALOG_UPLOAD_MSG="If you go back now , your upload image will be removed."
```
Comment (zoharHenMicrosoft) [inline app/src/main/java/com/example/cocktails/CustomItem/UserItemLevel1.kt:53]: need to remove space before the ","

File: app/src/main/java/com/example/cocktails/CustomItem/UserItemLevel1.kt [hunk 3]
```diff
@@ -45,18 +47,24 @@ class UserItemLevel1 : AppCompatActivity() {
...
     val EMPTY_FIELD_ERROR_MSG: String = "Field can not be empty"
     val MAX_LENGTH_COCKTAIL_NAME: Int = 12
     val COCKTAIL_ERROR_MSG_NAME = "Cocktail name already exist ,choose different name"
     val COCKTAIL_ERROR_MSG_LENGTH: String =
         "Cocktail name too long ,\n must me under $MAX_LENGTH_COCKTAIL_NAME characters"
```
Comment (zoharHenMicrosoft) [inline app/src/main/java/com/example/cocktails/CustomItem/UserItemLevel1.kt:58]: typo: "must be" instead "must me"

File: app/src/main/res/layout/filter_dialog.xml (hunk 1 above)
Comment (enavhasson) [reply app/src/main/res/layout/filter_dialog.xml:55]: yes , so weird hahahah. I will fix it

**Pull request merged**